recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-graphics/rcar-proprietary-graphic/rcar-proprietary-graphic.bb

```diff
@@ -11,9 +11,9 @@ PACKAGE_ARCH = "${MACHINE_ARCH}"
 S = "${WORKDIR}/xt-rcar"
 GLES = "gsx"
 
-SRC_URI_r8a7795 = "file://r8a7795-rcar-proprietary-graphic.tar.gz"
-SRC_URI_r8a7796 = "file://r8a7796-rcar-proprietary-graphic.tar.gz"
-SRC_URI_r8a77965 = "file://r8a77965-rcar-proprietary-graphic.tar.gz"
+SRC_URI_r8a7795 = "file://rcar-proprietary-graphic-${MACHINE}-domd.tar.gz"
+SRC_URI_r8a7796 = "file://rcar-proprietary-graphic-${MACHINE}-domd.tar.gz"
+SRC_URI_r8a77965 = "file://rcar-proprietary-graphic-${MACHINE}-domd.tar.gz"
```

Comment:
I don't see any difference in all three lines above

Reply:
Done